modules/installation-requirements-bare-metal.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-requirements-bare-metal-{context}"]
+= Bare metal machine requirements
+
+With a bare metal installation, you must deploy all of the required machines.
+
+[id="machine-requirements-{context}"]
+== Required machines
+
+The smallest {product-title} clusters require the following hosts:
+
+* 1 bootstrap machine
```

Comment:
Write in words numbers that are less than 10 per IBM Style

Reply:
Nevermind I think there can be an exception here, since numbers are presented in table later in the topic.